core/storage/base_model/gae_models_test.py

```diff
@@ -130,7 +130,7 @@ def test_generic_query_put_get_and_delete_operations(self) -> None:
         self.assertEqual(all_models.count(), 0)
         with self.assertRaisesRegexp(
             base_models.BaseModel.EntityNotFoundError,
-            'Entity for class BaseModel with id 4 not found'
+            'Entity for class BaseModel with id %s not found' % model_id
```

Comment:
Why is this changed / why was it working before?

Reply:
I think it wasn't working for some time, but I had some bugs in the code at that time. On the other hand I think the ID 4 is kinda random here and it is a bit hard to understand why it is 4 and not 5 for example.

Reply:
I would say keep it as '4' here, and have a self.assertEqual(model_id, '4') just before it.

In general, better to just compare to the exact thing.

Reply:
Done.